neuralteleportation/neuralteleportationmodel.py

```diff
@@ -55,6 +55,28 @@ def initialize_cob(self) -> None:
         """ Set the cob to ones. """
         self.teleport_activations(torch.ones(self.get_cob_size()))
 
+    def init_from_cumulative(self, cumulative, bins) -> None:
+        """
+            Initializes the weights of the network by a cumulative distribution.
+            Warning: it only works for
+        :param cumulative: cumulative distribution
+        :param bins: bins of histogram
+        :return: None
+        """
```

Comment:
This is the numpy docstyle, but throughout the rest of the code we use Google's docstyle. To remain consistent, I'd ajust this docstring to follow the usual style:
```suggestion
        """Initializes the weights of the network by a cumulative distribution.

        Notes:
            - This initialization method only works for
            
        Args:
            cumulative: cumulative distribution
            bins: bins of histogram
        """
```
P.S. The warning seems incomplete as well. And overall, the description doesn't really help much to understand what this method is doing, especially what its arguments mean.